tests/some-test-lines.R

```diff
@@ -0,0 +1,20 @@
+swe_set_ephe_path("C:\\ARCHAEOCOSMO\\ephe")
```

Comment:
This is problematic:
* `library(swephR)` is needed first
* the path is OS and environment dependent
Please keep in mind that code in the `test` directory is automatically executed during `R CMD check`